upup/pkg/kutil/rollingupdate_cluster.go

```diff
@@ -239,25 +246,42 @@ func buildCloudInstanceGroup(ig *api.InstanceGroup, g *autoscaling.Group, nodeMa
 	return n
 }
 
-func (n *CloudInstanceGroup) RollingUpdate(cloud fi.Cloud, force bool, interval time.Duration, k8sClient *release_1_5.Clientset) error {
+// Performs a rolling update on a list of nodes
+func (n *CloudInstanceGroup) RollingUpdate(cloud fi.Cloud, force bool, interval time.Duration, instanceGroupList *api.InstanceGroupList, k8sClient *release_1_5.Clientset) error {
 	c := cloud.(awsup.AWSCloud)
 
 	update := n.NeedUpdate
 	if force {
 		update = append(update, n.Ready...)
 	}
+
 	for _, u := range update {
-		instanceID := aws.StringValue(u.ASGInstance.InstanceId)
-		glog.Infof("Stopping instance %q in AWS ASG %q", instanceID, n.ASGName)
 
-		// TODO: Evacuate through k8s first?
+		drain, drainErr := NewDrainOptions(nil, u.Node.ClusterName)
```

Comment:
So in theory.  If you absolutely want me to do this I can.  Frankly, it makes my head hurt.  Do I need to do this?

We have some tricky logic situations, where I could have an `err` flow from one node to another.  I would have to carefully scope the `err`.  So carefully scope or have 3 variables instead of one variable name, and still actually still have three variables in mem?

What is the simplest this to do?